Automattic/wp-calypso
# [Odysseus] Like & dislike and open & close track event

Related to 
* https://github.com/Automattic/ai-services/issues/66 (merging this PR won't move the task to done)
* https://github.com/Automattic/ai-services/issues/75

## Proposed Changes

We are adding a basic thumbs up and down for liking and disliking messages. They will be only added to messages from the bot.
Also, we are firing the first Track event ever from Odysseus, open and close chat (visible / not visible)
Lastly, I've refactored a bit the Messages and extracted a single message to its own component, all along with its own styling

![image](https://github.com/Automattic/wp-calypso/assets/5689927/402a3203-8d6a-4de8-ab0e-944f4b609e7d)


## Testing Instructions

Apply this patch and assert that there are new buttons when the bot replies or start a conversation
Check that opening and closing the chat (Wapuu's ear) will fire these events:

![image](https://github.com/Automattic/wp-calypso/assets/5689927/63e6bc99-ecc7-40d6-82fe-6eb6d8ef5a7a)


## Pre-merge Checklist
